pageserver/src/tenant/storage_layer/layer.rs

```diff
@@ -1494,8 +1494,9 @@ impl LayerInner {
                 let duration = SystemTime::now().duration_since(local_layer_mtime);
                 match duration {
                     Ok(elapsed) => {
-                        let accessed = self.access_stats.accessed();
-                        if accessed {
+                        let accessed_and_visible = self.access_stats.accessed()
```

Comment:
Should we still bump the metric if these layers are evicted too early (sooner than their 2h threshold) or do we not care?

Reply:
I think it's fine b/c as long as the visibility algorithm is correct, these layers won't be accessed anyways.

Reply:
... so we don't need to care whether they are evicted or not

Reply:
They can still be accessed for other things (eg timestamp by lsn) but I don't think it matters much